### PR TITLE
Configure metal-ipi jobs for cluster-etcd-operator

### PR DESCRIPTION
**Pull request by ardaguclu**
This PR fixes misconfiguration for e2e-metal-ipi jobs and adds
two new blocking jobs namely e2e-metal-ipi-ovn-ipv6,
e2e-metal-ipi-serial-ipv4.

### CODE REVIEW

File: ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml [hunk 1]
```diff
@@ -1067,6 +1067,136 @@ presubmits:
...
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build01
+    context: ci/prow/e2e-metal-ipi-ovn-ipv6
+    decorate: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-cluster-etcd-operator-master-e2e-metal-ipi-ovn-ipv6
```
Comment (lilic) [inline ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml:1081]: Could you make this optional please?

File: ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml [hunk 2]
```diff
@@ -1067,6 +1067,136 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-metal-ipi,?($|\s.*)
+  - agent: kubernetes
+    always_run: true
```
Comment (lilic) [inline ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml:1071]: Not sure we need it to run everytime, so maybe we could start by making this false?

**Review by lilic — COMMENTED** [review]
Thanks!

File: ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml (hunk 1 above)
Comment (ardaguclu) [reply ci-operator/jobs/openshift/cluster-etcd-operator/openshift-cluster-etcd-operator-master-presubmits.yaml:1081]: Thanks for review @lilic. You are right, it is better to set them as optional. I'll change in this way.

**Review by lilic — APPROVED** [review]
/lgtm
/approve

Thank you!

**Pull request merged**